data/json/items/gun/40mm.json

```diff
@@ -180,5 +180,28 @@
     "clip_size": 3,
     "modes": [ [ "DEFAULT", "single", 1, "NPC_AVOID" ], [ "MULTI", "multi", 3, [ "NPC_AVOID", "SIMULTANEOUS" ] ] ],
     "proportional": { "weight": 1.5, "volume": 1.8, "price": 2 }
+  },
+  {
+    "id": "pseudo_m203",
+    "copy-from": "launcher_base",
+    "looks_like": "m203",
+    "type": "GUN",
+    "name": "M203 array",
+    "description": "Six M203 grenade launchers for use on an automated 40mm turret.",
```

Comment:
Should this get the "SEEING THIS IS A BUG" type description?

Reply:
I don't know how important that message is tbh, I can imagine situations where we might allow a player to see one of these